requirements.txt

```diff
@@ -1,6 +1,8 @@
+flake8
 lxml
 mock
 pep8
```

Comment:
do you still want to run pep8 alone? if not this can also be removed, and some code in the GNUmakefile too.